server.js

```diff
@@ -158,6 +161,30 @@ app.delete('/api/v1/playlists/:playlist_id/songs/:id', (request, response) => {
     })
 });
 
+app.get('/api/v1/playlists', (request, response) => {
```

Comment:
🎉So glad you got it. 

I'd recommend changing those `a` and `b` and `something` variable names to something more semantic.

Reply:
@bghalami